tfhe/src/core_crypto/fft_impl/fft64/crypto/bootstrap.rs

```diff
@@ -229,7 +229,7 @@ pub fn blind_rotate_scratch<Scalar>(
             // ct1 allocation
             StackReq::try_new_aligned::<Scalar>(glwe_size.0 * polynomial_size.0, CACHELINE_ALIGN)?,
             // external product
-            add_external_product_assign_scratch::<Scalar>(glwe_size, polynomial_size, fft)?,
+            cmux_scratch::<Scalar>(glwe_size, polynomial_size, fft)?,
```

Comment:
unnecessary as the code did not change, worst case rename the scratch function from the wop to have wop_blind_rotate_scratch and rename the associated function as well, this way we don't share scratch functions which are not related and will avoid updates to one piece of code breaking the other, wdyt ?